dspace/config/spring/rest/signposting.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+	<bean class="org.dspace.app.rest.signposting.converter.LinksetHttpMessageConverter"/>
+
+	<bean id="itemAuthorSignPosting" class="org.dspace.app.rest.signposting.relation.ItemAuthorProcessor">
+		<property name="metadataField" value="dc.contributor.author" />
+		<property name="relation" value="author" />
+		<property name="retrievedExternally" value="orcid" />
+		<property name="pattern" value="http://orcid.org/{0}" />
+	</bean>
+
+	<bean id="itemIdentifierSignPosting" class="org.dspace.app.rest.signposting.relation.ItemIdentifierProcessor">
+		<property name="metadataField" value="dc.identifier.doi" />
+		<property name="relation" value="cite-as" />
+		<property name="pattern" value="https://doi.org/{0}" />
+	</bean>
+
+	<bean id="itemPublicationBoundarySignPosting" class="org.dspace.app.rest.signposting.relation.ItemPublicationBundaryProcessor">
+		<property name="pattern" value="{0}/{1}/{2}" />
+	</bean>
+
+	<bean id="bitstreamPublicationBundarySignPosting" class="org.dspace.app.rest.signposting.relation.BitstreamPublicationBundaryProcessor" />
+
+	<bean id="bitstreamLinksetProcessor" class="org.dspace.app.rest.signposting.relation.BitstreamLinksetProcessor" />
+
+	<bean id="bitstreamTypeProcessor" class="org.dspace.app.rest.signposting.relation.BitstreamTypeProcessor" >
+		<property name="metadataField" value="" />
```

Comment:
Why are all the `metadataField` properties in this XML configuration empty?  Are they supposed to have values?  If not, then why do we have these other Processors?  How can we add more accurate tests for the `*LicenseProcessor` classes and `*TypeProcessor` classes?

Reply:
signposting.xml has been cleaned up.
Additional tests has been added.